### PR TITLE
feat: Update container level operations.

### PR DESCRIPTION
**Pull request by prmukherj**
1. Emulated the settings dict-style version (example shown in test.)
2. Display name attached to tasks.

### CODE REVIEW

File: tests/test_meshing_workflow.py [hunk 1]
```diff
@@ -714,3 +714,35 @@ def test_fault_tolerant_workflow(exhaust_system_geometry, new_mesh_session):
...
+        "Update Boundaries",
+        "Create Regions",
+        "Update Regions",
+        "Add Boundary Layers",
+        "Generate the Volume Mesh",
+    }
+
+    task_display_name = []
+    for task in meshing.workflow.TaskObject:
+        task_display_name.append(task.display_name())
```
Comment (seanpearsonuk) [inline tests/test_meshing_workflow.py:740]:
```suggestion
    task_display_names = []
    for task in meshing.workflow.TaskObject:
        task_display_names.append(task.display_name())
```

File: tests/test_meshing_workflow.py [hunk 2]
```diff
@@ -714,3 +714,35 @@ def test_fault_tolerant_workflow(exhaust_system_geometry, new_mesh_session):
...
+
+    assert set(task_display_name) == task_object_display_names
```
Comment (seanpearsonuk) [inline tests/test_meshing_workflow.py:742]:
```suggestion
    assert set(task_display_names) == task_object_display_names
```

File: tests/test_meshing_workflow.py [hunk 3]
```diff
@@ -714,3 +714,35 @@ def test_fault_tolerant_workflow(exhaust_system_geometry, new_mesh_session):
...
+
+    task_display_name = []
+    for name, _ in meshing.workflow.TaskObject.items():
+        task_display_name.append(name)
+
+    assert set(task_display_name) == task_object_display_names
```
Comment (seanpearsonuk) [inline tests/test_meshing_workflow.py:748]:
```suggestion
    task_display_names = []
    for name, _ in meshing.workflow.TaskObject.items():
        task_display_names.append(name)

    assert set(task_display_names) == task_object_display_names
```

File: src/ansys/fluent/core/workflow.py [hunk 4]
```diff
@@ -381,6 +384,9 @@ def __dir__(self):
             )
         )
 
+    def items(self):
+        return self._task_container.get_state().items()
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/workflow.py:388]: So in fact, datamodel is fine, issues are at workflow only.

File: src/ansys/fluent/core/workflow.py [hunk 5]
```diff
@@ -333,6 +333,9 @@ def _tasks_with_matching_attributes(self, attr: str, other_attr: str) -> list:
                 matches.append(task)
         return matches
 
+    def display_name(self):
+        return self.get_id()
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/workflow.py:337]: Something might be awry here. E.g., Display name would be "Import Geometry" for id "TaskObject1".

**Review by seanpearsonuk — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**